Microsoft.Toolkit.Uwp.UI.Media/Effects/Extensions/BackdropSourceExtension.cs

```diff
@@ -4,28 +4,30 @@
 
 using System;
 using Microsoft.Toolkit.Uwp.UI.Media.Pipelines;
+using Windows.UI.Xaml.Markup;
 using Windows.UI.Xaml.Media;
 
 namespace Microsoft.Toolkit.Uwp.UI.Media.Effects
 {
     /// <summary>
     /// A backdrop effect that can sample from a specified source
     /// </summary>
-    public sealed class BackdropEffect : IPipelineInput
+    [MarkupExtensionReturnType(ReturnType = typeof(PipelineBuilder))]
+    public sealed class BackdropSourceExtension : MarkupExtension
     {
         /// <summary>
         /// Gets or sets the backdrop source to use to render the effect
         /// </summary>
-        public AcrylicBackgroundSource Source { get; set; }
+        public AcrylicBackgroundSource BackgroundSource { get; set; }
```

Comment:
Make the default for this `Backdrop` so it doesn't have to be as verbose unless you want HostBackdrop?

Reply:
Ah, good idea! This will also make the default value consistent with the one from the official [`AcrylicBrush`](https://docs.microsoft.com/uwp/api/windows.ui.xaml.media.acrylicbrush), which defaults to `Backdrop` as well for this parameter. The weird thing though is that their `enum` does have `HostBackdrop` as initial value, but apparently they manually initialize the default parameter to `Backdrop`. I guess they should call that out in the docs as well 🤔

Fixed in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3298/commits/43989a133dded55c58b9ad8d504d20348c3c4ce1.